src/hm-admin/services/core/CoreDefinition.service.ts

```diff
@@ -0,0 +1,95 @@
+
+import {Http} from '@angular/http';
+import {Injectable} from '@angular/core';
+import 'rxjs/add/operator/map';
+import {Observable} from 'rxjs';
+
+@Injectable()
+export class CoreDefinition {
+
+    public coreDefinition: any;
+
+    constructor(private http: Http) {}
+
+    /**
+     * Get the CoreDefinition, if it exists from Localstorage, if not from HTTP
+     *
+     * @param {string} url
+     *
+     * @returns {Observable}
+     */
+    getCoreDefinition(url) {
+        let definition;
+        this.coreDefinition = this.getCacheCoreDefinition();
+
+        if (this.coreDefinition === null) {
+            definition = this.fetchCoreDefinition(url);
+            this.setCacheCoreDefinition(definition);
+        }
+        return this.coreDefinition;
```

Comment:
I think it warn you that you have trailing space which is not the same (if you have useless space of tab on a line, it's trailing space)